src/main/sensors/initialisation.c

```diff
@@ -84,6 +87,9 @@ bool sensorsAutodetect(void)
     baroInit();
 #endif
 
+positionInit();
+gpsRescueInit();
+
```

Comment:
Instead of from here it's probably better/safer to call these init functions from `main/fc/init.c`.

-----

### Add `gpsRescueInit()` at approx. line 765:
```c
#ifdef USE_GPS
    if (featureIsEnabled(FEATURE_GPS)) {
        gpsInit();
        gpsRescueInit();
    }
#endif
```

-----

### Add `altitudeInit()` at approx. line 827:
```c
#ifdef USE_BARO
    // baro got initialized in sensorsAutodetect()
    baroStartCalibration();
    altitudeInit();
#endif
```

Reply:
thanks, done